infrastructure/main.tf

```diff
@@ -47,13 +47,13 @@ module "wa_task_management_api_database_flexible" {
     azurerm.postgres_network = azurerm.postgres_network
   }
 
-  source             = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
-  product            = var.product
-  component          = var.component
-  name               = "${var.postgres_db_component_name}-postgres-db-flexible"
-  location           = var.location
-  business_area      = var.business_area
-  env                = var.env
+  source        = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+  product       = var.product
+  component     = var.component
+  name          = "${var.postgres_db_component_name}-postgres-db-flexible"
+  location      = var.location
+  business_area = var.business_area
+  env           = var.env
   pgsql_databases = [
```

Comment:
seems to be just formatting changes here?  Can you undo those please?

Reply:
Looks like the bot automatically formatted it this way. https://github.com/hmcts/wa-task-management-api/pull/1279/commits/856534032cc781fb54d3a7cb5f1784fe08efce2c

Reply:
That's annoying it's made it worse

Reply:
Can leave as is in that case.